oss_src/unity/lib/unity_sframe.cpp

```diff
@@ -323,6 +323,23 @@ size_t unity_sframe::column_index(const std::string &name) {
   return sf->column_index(name);
 }
 
+std::string unity_sframe::column_name(size_t index) {
+  Dlog_func_entry();
+
+  auto sf = this->get_underlying_sframe();
```

Comment:
This can be done without get_underlying_sframe. This will force materialization to get a column name.